Aztec/Classes/TextKit/TextView.swift

```diff
@@ -724,6 +767,56 @@ open class TextView: UITextView {
     }
 
 
+    /// Removes the List Attributes from a collection of attributes, whenever:
```

Comment:
I'd consider adding extra documentation to explain why this is necessary.  Specifically something like:

```swift
// This is necessary because when the caret is at EOF, and the previous `\n` character has
// a textList style, that style will remain in the `typingAttributes`.  We'll only allow the style
// to remain if there are contents in the current line with the textList style (in which case
// this condition won't ever trigger because we'll either no longer be at EOF, or the previous
// character won't be `\n`).